File: day_seven/multiples.cpp
#include <iostream>

int main() {
    int num = 1;

    do {
        if (num % 3 == 0 && num % 5 == 0) {
            std::cout << num << std::endl;
        }
        num = num + 1;
    } while (num < 1000);
    return 0;
}

File: day_seven/multiples.py
#!/usr/bin/python3

import sys

def multiples():

    for num in range(1, 1000+1):
        if num % 3 == 0 and num % 5 == 0:
            print(num)

def main():
    multiples()

if __name__=="__main__":
    main()